src/nft/css/common.css.ts

```diff
@@ -28,7 +28,7 @@ const magicalGradient = style({
       content: '',
       position: 'absolute',
       inset: '-1px',
-      background: 'linear-gradient(91.46deg, #4673FA 0%, #9646FA 100.13%) border-box',
+      background: 'linear-gradient(45deg, #FC72FF40 0%, #FC72FF20 100.13%) border-box',
```

Comment:
non-blocking: border is a very faint so it's hard to notice imo
![Screenshot 2023-09-12 at 9 41 40 AM](https://github.com/Uniswap/interface/assets/11512321/87791d53-327c-4107-883b-1d5e7779e3f5)
![Screenshot 2023-09-12 at 9 42 12 AM](https://github.com/Uniswap/interface/assets/11512321/57b32d10-6c26-4563-af7c-55b9294afc8c)

Reply:
good point - brought to 80% opacity